api/orchestration.go

```diff
@@ -84,7 +87,7 @@ func (o *docDBOrchestrator) documentDBCreate(ctx context.Context, req *DocDBCrea
 
 		msgChan <- fmt.Sprintf("requested creation of docdb cluster %s", cl)
 
-		if err = retry(10, 2*time.Second, func() error {
+		if err = retry(10, 3, 10*time.Second, func() error {
```

Comment:
OK, so this should go 10, 20, 40 seconds, and then stop increasing the retry interval.

Reply:
it will sleep for 10, 20, 40, 80, 80, 80, ...

Reply:
nice